pkg/api/agentPoolOnlyApi/v20180331/validate.go

```diff
@@ -39,7 +39,7 @@ func handleValidationErrors(e validator.ValidationErrors) error {
 	ns := err.Namespace()
 	switch ns {
 	// TODO: Add more validation here
-	case "Properties.LinuxProfile", "Properties.ServicePrincipalProfile.ClientID",
+	case "Properties.ServicePrincipalProfile.ClientID",
```

Comment:
please add a UT to verify this change

Reply:
Synced up offline, there is no need to write UT for this change